packages/s3-request-presigner/README.md

```diff
@@ -91,4 +91,46 @@ to `presigned` is not sufficient to make a request. You need to send the
 server-side encryption headers along with the url. These headers remain in the
 `presigned.headers`
 
+getSignedUrl Example:
+
+```javascript
+import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
```

Comment:
The example should be prettified using [prettier playground](https://prettier.io/playground/)

Reply:
Formatted